tools/viewer/AnDOViewer.py

```diff
@@ -0,0 +1,97 @@
+import pandas as pd
+import argparse
+import os
+import sys
+
+def show_struct(directory):
+    """
+    Show the structure of the directory given in  argument
+
+    Args:
+        directory ([Path]): [Path of the directory to show]
+    """
+    cmd = "tree -d  "+directory
+    os.system(cmd)
+
+
+def show_experiments(directory):
+    """
+    Show the experiments in the directory given in  argument
+
+    Args:
+        directory ([Path]): [Path of the directory to show]
+    """
+    cmd = "tree -d -L 1 "+directory
+    os.system(cmd)
+
+
+def show_subjects(directory):
+    """
+    Show the subjects in the directory given in  argument
+
+    Args:
+        directory ([Path]): [Path of the directory to show]
+    """
+    cmd = "tree -d -L 2 "+directory
+    os.system(cmd)
+
+
+def show_sessions(directory):
+    """
+    Show the sessions in the directory given in  argument
+
+    Args:
+        directory ([Path]): [Path of the directory to show]
+    """
+    cmd = "tree -d -L 3 "+directory
+    os.system(cmd)
+
+
+def main():
+    """
+    usage: AnDO_Viewer.py [-h] [-S] [-Se] [-Su] [-Ss] pathToDir
```

Comment:
same... please unify your use of underscores in the files / tools names!

Reply:
Done